src/index.tsx

```diff
@@ -1,13 +1,36 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { PageNotFound } from './components/PageNotFound';
+import { GlobalContextProvider } from './store/GlobalContextProvider';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <GlobalContextProvider>
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+
+          <Route path="home" element={<Navigate to={'/'} />} />
+
+          <Route path="people" element={<PeoplePage />}>
+            <Route path=":slug?" element={<PeoplePage />} />
```

Comment:
The nested route `<Route path=":slug?" element={<PeoplePage />} />` inside the `people` route is redundant since `PeoplePage` is already the element for the `people` route. Consider removing the nested route to simplify the routing configuration.